io.github.fheroes2.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
```

Comment:
this file exist in the repository. you should use it instead of this one.

albeit it doesn't seem to be installed by the upstream buildsystem, which is a bug.

Reply:
https://github.com/ihhub/fheroes2/pull/5405

Reply:
but now it's gone from here which mean the build will fail.